public-api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -20,7 +35,7 @@
   </system-properties>
 
   <automatic-scaling>
-    <min-idle-instances>${GAE_MIN_IDLE_INSTANCES}</min-idle-instances>
-    <max-instances>${GAE_MAX_INSTANCES}</max-instances>
+    <min-idle-instances>1</min-idle-instances>
+    <max-instances>10</max-instances>
```

Comment:
If those numbers come from RW. I don't know the traffic for data browser so the same number might not be the best for data brower. Worth to check GAE metrics to get what is the machine, instance count you need.